src/esp/assets/ResourceManager.cpp

```diff
@@ -180,31 +188,85 @@ bool ResourceManager::loadStage(
       ((_physicsManager != nullptr) &&
        (_physicsManager->getInitializationAttributes()->getSimulator().compare(
             "none") != 0));
-  const Magnum::ResourceKey& renderLightSetup(stageAttributes->getLightSetup());
+  const std::string renderLightSetupKey(stageAttributes->getLightSetup());
   std::map<std::string, AssetInfo> assetInfoMap =
       createStageAssetInfosFromAttributes(stageAttributes, buildCollisionMesh,
                                           loadSemanticMesh);
 
+  // set equal to current Simulator::activeSemanticSceneID_ value
```

Comment:
This is not new code. I've simply moved the semantic loading to happen earlier in this function. I did this because I need to know if a render asset instance is going to be used for RGBD/Semantic/both at the time of creation. I want to know as early as possible whether we're going to be in "single scene graph" vs "dual scene graph" mode.